python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -932,6 +937,10 @@ def run_retries_enabled(self) -> bool:
     def run_retries_max_retries(self) -> int:
         return self.get_settings("run_retries").get("max_retries", 0)
 
+    @property
+    def retry_on_asset_or_op_failure(self) -> bool:
```

Comment:
if this is going to be a top-level instance field i think it is worth clarifying that it's about run retries in the name

Also cloud accesses this field in a different way than get_settings() (it looks at deployment settings) so we will need to account for that / override it if needed)

We'll want to make sure that we test this change on cloud as well as OSS

Reply:
for future traceability - see https://github.com/dagster-io/internal/pull/12763 https://github.com/dagster-io/dagster/pull/26046